why-oc.md

```diff
@@ -57,6 +71,107 @@ So now lets take a look at OpenCore's method:
+
+For example, OpenCore goes through proper security audits to ensure it complies with UEFI Secure Boot, and is the only Hackintosh bootloader to undergo these rigorous reviews and have such support.
+
+Version 0.6.1 was originally designed to be the official release of OpenCore as it would have proper UEFI/Apple Secure Boot, and would be the 1 year anniversary of OpenCore's release as a public tool. However, due to circumstances around macOS Big Sur and the rewriting of OpenCore's prelinker to support it, it was decided to push of 1.0.0 for another year.
+
+Current road map:
+
+* 2019: Year of Beta
+* 2020: Year of Secure Boot
+* 2021: Year of Refinement
+
+So please do not see the version number as a hindrance, instead as something to look forward to.
+
+## Does OpenCore always inject SMBIOS and ACPI data into other OSes
+
+By default, OpenCore will assume that all OSes should be treated equally in regards to ACPI and SMBIOS information. The reason for this thinking is 3 parts:
+
+* This allows for proper multiboot support like with [BootCamp](https://dortania.github.io/OpenCore-Post-Install/multiboot/bootcamp.html)
+* Avoids poorly made DSDTs and encourages proper ACPI practices
+* Avoids edge cases where info is injected several times, commonly seen with Clover
+  * ie. How would you handle SMBIOS and ACPI data injection once you booted boot.efi, but then get kicked out? The changes are already in memory and so trying to undo them can be quite dangerous. This is why Clover's method is frowned upon
+
+However, there are quirks in OpenCore that allow for SMBIOS injection to be macOS limited by patching where macOS reads SMBIOS info from. The `CustomSMIOSGuid` quirk with `CustomSMBIOSMode` set to `Custom` can break in the future and so we only recommend this option in the event of certain software breaking in other OSes. For best stability, please disable these quirks.
+
+## Does OpenCore require a fresh install
+
+Not at all in the event you have a "Vanilla" installation, what this refers to is whether the OS has tampered in any way such as installing 3rd party kexts into the system volume or other unsupported modifications by Apple. When your system has been heavily tampered either by you or 3rd party utilities like Hackintool, we recommend a fresh install to avoid any potential issues
+
+Special note for Clover users, please reset your NVRAM when installing with OpenCore. Many of Clover variables can conflict with OpenCore and macOS.
+
+## Does OpenCore only support limited versions of macOS
+
+As of OpenCore 0.6.2, you can now boot every Intel version of macOS going all the way back to OS X 10.4! Proper support however will depend on your hardware, so please verify yourself: [Hardware Limitations](macos-limits.md)
+
+::: details macOS Install Gallery
+
+We've tested many versions here at Acidanthera, and I myself have run many versions of OS X on my old HP DC 7900(Core2 Quad Q8300). Here's just a small gallery of what I've tested:
```

Comment:
```suggestion
Acidanthera has tested many versions, and I myself have run many versions of OS X on my old HP DC 7900(Core2 Quad Q8300). Here's just a small gallery of what I've tested:
```

Reply:
Smh I am Acidanthera, *stop hiding the truth*

Reply:
I am not Acidanthera smh